src/home/ExpandedCard.js

```diff
@@ -22,14 +43,15 @@ const ExpandedCard = ({ navigation }) => {
       <ScrollView>
         <SafeAreaView
           style={{ justifyContent: 'center', alignItems: 'center' }}>
-          {/* pass in props.cardTitle eventually */}
-          <Text style={styles.title}> TITLE </Text>
+          <Text style={styles.title}> {title} </Text>
 
           <SafeAreaView style={styles.box}>
-            {/* CARD CONTENT GOES HERE */}
+            <Image
+              source={{ uri: `file://${cardImagePath}` }}
+              style={styles.cardImage}
+              resizeMode="contain"
+            />
```

Comment:
```suggestion
{ cardImagePath && 
            (<Image
              source={{ uri: `file://${cardImagePath}` }}
              style={styles.cardImage}
              resizeMode="contain"
            />)
}
```
Only loads image if `cardImagePath` is not empty.